### ISSUE TITLE
 'os' has no attribute 'sched_getaffinity'

### ISSUE THREAD

**Issue by jeffg2k** [JohnHammond/katana#8]
It would be nice if in the manager.py there was an exception value for `threads` as os.sched_getaffinity is not available on all nix systems.  I'm running macOS for example and this fails.  If I set it to a static value, it runs fine.
```
  File "/Users/Jeff/Documents/workspace/katana/env/bin/katana", line 11, in <module>
    load_entry_point('katana==2.0', 'console_scripts', 'katana')()
  File "/Users/Jeff/Documents/workspace/katana/env/lib/python3.7/site-packages/katana-2.0-py3.7.egg/katana/__main__.py", line 36, in main
    manager = Manager(monitor=monitor, config_path=args.config)
  File "/Users/Jeff/Documents/workspace/katana/env/lib/python3.7/site-packages/katana-2.0-py3.7.egg/katana/manager.py", line 70, in __init__
    "threads": len(os.sched_getaffinity(0)),
AttributeError: module 'os' has no attribute 'sched_getaffinity'
```

**Comment by calebstewart** [comment]
I added a try/except for the default. I'd prefer `sched_getaffinity` as it properly reports if the process is restricted in some way. The branch will fallback to using `multiprocessing.cpu_count()` if it isn't available, though. To my understanding, `multiprocessing.cpu_count` should be available everywhere.

Please let me know if that doesn't fix the issue. I was going to check back before closing the issue, but github closed it automatically because I said "should fix #8" in my commit message, haha.

**Comment by calebstewart** [comment]
Since I didn't hear back, I'm going to close this issue, as the problem appears to be fixed from my end.
